### PR TITLE
Catch deleted xforms error

### PR DESCRIPTION
**Pull request by esoergel**
http://manage.dimagi.com/default.asp?186595
@nickpell

### CODE REVIEW

File: corehq/ex-submodules/couchforms/__init__.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
-from .exceptions import XMLSyntaxError, CouchFormException
+from .exceptions import XMLSyntaxError, CouchFormException, UnexpectedDeletedXForm
```
Comment (dimagimon) [inline corehq/ex-submodules/couchforms/__init__.py]: F401 'CouchFormException' imported but unused

Comment (dimagimon) [inline corehq/ex-submodules/couchforms/__init__.py]: F401 'UnexpectedDeletedXForm' imported but unused

Comment (dimagimon) [inline corehq/ex-submodules/couchforms/__init__.py]: F401 'XMLSyntaxError' imported but unused

**Comment by nickpell** [comment]
looks good when tests pass

**Comment by snopoke** [comment]
Build should get fixed by https://github.com/dimagi/commcare-hq/pull/9087

**Comment by czue** [comment]
:boot: ed the :construction:

**Pull request merged**